src/main/java/goodgenerator/crossmod/nei/ComponentAssemblyLineHandler.java

```diff
@@ -0,0 +1,158 @@
+package goodgenerator.crossmod.nei;
+
+import codechicken.nei.recipe.GuiCraftingRecipe;
+import codechicken.nei.recipe.GuiUsageRecipe;
+import codechicken.nei.recipe.TemplateRecipeHandler;
+import cpw.mods.fml.common.event.FMLInterModComms;
+import gregtech.GT_Mod;
+import gregtech.api.enums.GT_Values;
+import gregtech.api.util.GT_Recipe;
+import gregtech.api.util.GT_Utility;
+import gregtech.common.power.EUPower;
+import gregtech.common.power.Power;
+import gregtech.common.power.UnspecifiedEUPower;
+import gregtech.nei.GT_NEI_DefaultHandler;
+import net.minecraft.util.EnumChatFormatting;
+
+import java.awt.*;
+
+public class ComponentAssemblyLineHandler extends GT_NEI_DefaultHandler {
+    public ComponentAssemblyLineHandler(GT_Recipe.GT_Recipe_Map aRecipeMap) {
+
+        super(aRecipeMap);
+        this.transferRects.remove(new RecipeTransferRect(new Rectangle(65, 13, 36, 18), getOverlayIdentifier()));
+        this.transferRects.add(new RecipeTransferRect(new Rectangle(69, 18, 9, 34), getOverlayIdentifier()));
+        if (!NEI_Config.isAdded) {
+            FMLInterModComms.sendRuntimeMessage(
+                GT_Values.GT,
+                "NEIPlugins",
+                "register-crafting-handler",
+                "gregtech@" + this.getRecipeName() + "@" + this.getOverlayIdentifier());
+            GuiCraftingRecipe.craftinghandlers.add(this);
+            GuiUsageRecipe.usagehandlers.add(this);
+        }
+    }
+    @Override
+    public TemplateRecipeHandler newInstance() {
+        return new ComponentAssemblyLineHandler(this.mRecipeMap);
+    }
+
+    @Override
+    public void drawExtras(int aRecipeIndex) {
+        GT_Recipe recipe = ((CachedDefaultRecipe) this.arecipes.get(aRecipeIndex)).mRecipe;
+        drawDescription(recipe);
+    }
+
+    // I had to do this because the original method was private, so I couldn't override it.
```

Comment:
You can override `getSpecialInfo` to format special value. Look at `GT_NEI_FluidReactor` in GT++.
(Yes, I agree, the way GT formats special values is quite a mess)

Reply:
fixed in [822eaaa](https://github.com/GTNewHorizons/GoodGenerator/commit/822eaaa98f6679c947d674ef4ee74f318c31597e)